<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollectionSettingsPage</class>
 <widget class="QWidget" name="CollectionSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Collection</string>
  </property>
  <layout class="QVBoxLayout" name="layout_collectionsettingspage">
   <item>
    <widget class="QLabel" name="label_collection_folders">
     <property name="text">
      <string>These folders will be scanned for music to make up your collection</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_collection_folders">
     <item>
      <widget class="QListView" name="list">
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layout_collection_folder_buttons">
       <item>
        <widget class="QPushButton" name="add">
         <property name="text">
          <string>Add new folder...</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="remove">
         <property name="text">
          <string>Remove folder</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_collection_buttons">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_updating">
     <property name="title">
      <string>Automatic updating</string>
     </property>
     <layout class="QVBoxLayout" name="layout_updating">
      <item>
       <widget class="QCheckBox" name="startup_scan">
        <property name="text">
         <string>Update the collection when Strawberry starts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="monitor">
        <property name="text">
         <string>Monitor the collection for changes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="live_scanning">
        <property name="text">
         <string>Use live scanning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="prevent_delete">
        <property name="text">
         <string>Never delete songs from the collection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_preferred_cover_filenames">
        <property name="text">
         <string>Preferred album art filenames (comma separated)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cover_art_patterns">
        <property name="toolTip">
         <string>When looking for album art Strawberry will first look for picture files that contain one of these words.
If there are no matches then it will use the largest image in the directory.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_display">
     <property name="title">
      <string>Display options</string>
     </property>
     <layout class="QVBoxLayout" name="layout_display">
      <item>
       <widget class="QCheckBox" name="auto_open">
        <property name="text">
         <string>Automatically open single categories in the collection tree</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="pretty_covers">
        <property name="text">
         <string>Show album cover art in collection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="show_dividers">
        <property name="text">
         <string>Show dividers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_albumcovers">
     <property name="title">
      <string>Saving album covers</string>
     </property>
     <layout class="QVBoxLayout" name="layout_albumcovers">
      <item>
       <widget class="QCheckBox" name="checkbox_cover_album_dir">
        <property name="text">
         <string>Save album covers in album directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupbox_cover_filename">
        <property name="title">
         <string>Filename:</string>
        </property>
        <layout class="QVBoxLayout" name="layout_cover_filename">
         <item>
          <widget class="QWidget" name="groupbox_cover_type" native="true">
           <layout class="QHBoxLayout" name="layout_cover_type">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="radiobutton_cover_hash">
              <property name="text">
               <string>Use hash</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radiobutton_cover_pattern">
              <property name="text">
               <string>Use pattern</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer_cover_filename_bttons">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineedit_cover_pattern">
           <property name="text">
            <string>%albumartist-%album</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkbox_cover_overwrite">
           <property name="text">
            <string>Overwrite existing file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkbox_cover_lowercase">
           <property name="text">
            <string>Lowercase filename</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkbox_cover_replace_spaces">
           <property name="text">
            <string>Replace spaces with dashes</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>list</tabstop>
  <tabstop>add</tabstop>
  <tabstop>remove</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
